process/pyladies-projects-overview.md

```diff
@@ -0,0 +1,51 @@
+# PyLadies Projects
+
+The PyLadies Project teams were kick started in an effort to provide other means of participation for PyLadies members while also increasing the ability of PyLadies to manage itself as an open source project. See the [Request for Comment issue](https://github.com/pyladies/global-organizing/issues/33) for information.
+
+## Contributing to a Project 
+
+Any self-identifying PyLadies member can participate as a contributing member. Additionally each project will require a project lead or project leads. Any project member can self-nominate or nominate another person to be a project lead. If nominating another to be a project lead, the individual must be a self-identifying PyLadies member and their consent will be required (e.g. an email sent to the person and they consent to participate).
+
+Membership as a PyLadies member, therefore, will rely on self-identification.
+
+Additional requirements to consider:
+
+- Basic communication in English or ability to alternatively contribute to the project team through English (e.g. a local volunteer can help include updates in English representing volunteers from a geographic region). Please note that the language(s) a Project team can support depend on the members of the team, if additional languages are needed please note in your proposal (see below) or contact that project team directly.
+- Reasonably responsive on email, GitHub, and Slack
+
+## To start a project
+
+1. Open up a new issue outlining the scope of the project, project team members information, and other requested details with [this template](.github/ISSUE_TEMPLATE/project-proposal.md). 
+2. Solicit feedback and members in the [PyLadies Slack](https://slackin.pyladies.com/) using the channels #general and #project-proposals.
+3. Get approval from the [PyLadies Global Admin Team](https://github.com/orgs/pyladies/teams/pyladies-global-admin).
```

Comment:
This links is probably 404 if they're not a member of the PyLadies org, so they can't view the list of global admin team members.
Perhaps we can list the names elsewhere, or instead just tag `@pyladies/pyladies-global-admin`

Reply:
Good point thanks for that!

Reply:
Actually that ought to be template change since I don't think it's on there.